site/docs/components/fieldset.mdx

```diff
@@ -0,0 +1,264 @@
+---
+name: Fieldset
+menu: Components
+route: /components/fieldset
+---
+
+import { Playground } from 'docz';
+import { Fieldset, StatusLabel, SelectionGroup, RadioButton, TextInput } from 'hds-react';
+
+import LargeParagraph from '../../src/components/LargeParagraph';
+import Link from '../../src/components/Link';
+
+
+# Fieldset
+
+<StatusLabel type="alert">Draft</StatusLabel>
+<StatusLabel type="success" style={{ marginLeft: 'var(--spacing-xs)' }}>
+  Accessible
+</StatusLabel>
+
+<LargeParagraph>Fieldset groups multiple related form input fields together by giving them a common legend heading.</LargeParagraph>
+
+## Principles
+
+- The Fieldset component should only contain form elements such as input fields and form controls. See [HDS Form pattern documentation](/patterns/forms) for more information about available form elements.
+  - It is recommended that Fieldsets always contain at least two (2) form elements.
+- As a rule of thumb, if a group of form elements require a heading, the usage of a Fieldset is recommended.
+  - For example, multiple input fields related to inputting user's address can be grouped together with a Fieldset with a heading "_Address_".
+- A heading must be given to the Fieldset. The heading should be clear and describe the input group.
+  - A good heading desribes why the form element inside the Fieldset are grouped together. For example, a heading "_Guardian information_" tells the user that the following elements are meant to input personal information related to child's guardian.
+- **HDS Fieldset does not style the content in any way.** Depending on the layout of the service, you may have to add space between the inputs to distinguish them from each other.
```

Comment:
This is true except the disabled attribute might add some styling to the content? https://developer.mozilla.org/en-US/docs/Web/HTML/Element/fieldset#attr-disabled